.travis.yml

```diff
@@ -9,9 +9,10 @@ dist: trusty
 jobs:
   include:
     - &analyze_and_format
-      stage: analyze/format
+      stage: analyze_and_format
       dart: dev
       script: ./tool/travis.sh dartanalyzer dartfmt
+      if: env(PKG) IS present
```

Comment:
when is it not?

Reply:
For this very first one - I think its weird that it tries to run this one at all but we could just set the first PKG here as well. It just bothered me so I did this instead.